README.md

```diff
@@ -77,6 +77,17 @@ Then we have an automated six step deployment process, which runs in CircleCI.
 
 Our staging and production environments are hosted by AWS. We would deploy to production per each feature/config merged into  `master`  branch.
 
+## Static Code Analysis
```

Comment:
This is a good README section: concise but clear and informative 👍

Reply:
Thanks! 😁